src/main/scala/pesto/http.scala

```diff
@@ -0,0 +1,19 @@
+import io.circe.{Decoder}
+import cats.effect._
+import org.http4s.client.blaze._
+import org.http4s.circe._
+import scala.concurrent.ExecutionContext.global
+import org.http4s.Uri
+import org.http4s.client.middleware.FollowRedirect
+
+package pesto {
+  package object http {
+    implicit val cs: ContextShift[IO] = IO.contextShift(global)
+    implicit val timer: Timer[IO] = IO.timer(global)
+
+    val client = BlazeClientBuilder[IO](global).resource
+
+    def fetch[A: Decoder](uri: Uri): IO[A] = client.use(c => FollowRedirect(10)(c).expect(uri)(jsonOf[IO, A]))
```

Comment:
This will create a new client for every invocation of fetch. That probably isn't what you would want in production.

Reply:
so `use` is the bracketing construct? I was thinking resource would be, and use just exposes the controlled resource.

Reply:
There is a different variant called allocated where you get the resource and a cleanup IO, but that is harder to manage.

Reply:
So are you saying I should use the stream constructor in this case?

Reply:
It doesn't necessarily have to be the stream construct, its just that it isn't designed to be instantiated for every request.